engine/crates/parser-postgresql/src/registry/types/page_info.rs

```diff
@@ -0,0 +1,41 @@
+use crate::registry::context::{InputContext, OutputContext};
+use common_types::auth::Operations;
+use engine::registry::{resolvers::transformer::Transformer, MetaField, ObjectType};
+
+pub(super) fn register(input_ctx: &InputContext<'_>, output_ctx: &mut OutputContext) {
+    let object = ObjectType::new(
+        input_ctx.type_name("PageInfo"),
+        [
+            MetaField {
+                name: "hasPreviousPage".to_string(),
+                ty: "Boolean!".into(),
+                resolver: Transformer::PaginationData.and_then(Transformer::select("has_previous_page")),
```

Comment:
Probably not that important, but seems odd we don't put this `PaginationData` resolver onto the `pageInfo` field and save having to do it for every single field.

Though maybe there's a reason that won't work...

Reply:
Yeah, I tried that. So we then don't have the pagination data in the last resolved value, just the data from the select is available (if I just keep the select here, move the pagination data to the `pageInfo`).

I can also just try to select the whole object at a time for `pageInfo`, but the default resolver for these fields is then `Parent` and it tries to convert the `PageInfo` object into a boolean etc. later on which breaks.